extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/QuarkusErrorHandler.java

```diff
@@ -53,7 +53,7 @@ public void handle(RoutingContext event) {
                     public void run() {
                         event.response().end();
                     }
-                });
+                }).subscribeAsCompletionStage();
```

Comment:
Not totally sure what you are doing here.
The subscription will trigger the computation.
But you may want to be sure that an error has not been emitted.